config/spryk/spryks/Spryker/Zed/Business/AddCrudFacade.yml

```diff
@@ -269,9 +269,17 @@ postSpryks:
                       value: "is_active"
                   propertyType:
                       value: "BOOLEAN"
-
     # Add add{{ domainEntity }}DataBuilderDefinition for the tests transfer builder
     -   AddDataBuilderDefinition:
             arguments:
                 name:
                     value: "{{ domainEntity }}"
+    # Add Spy{{ domainEntity }}Query method to persistence factory
+    -   AddZedDomainEntityPersistenceFactoryMethod:
+            arguments:
+                fqcnClassName:
+                    value: "\\Orm\\Zed\\{{ module }}\\Persistence\\Spy{{ domainEntity }}Query"
+                factoryMethod:
+                    value: "create{{ domainEntity }}Query"
+                output:
+                    value: "\\Orm\\Zed\\{{ module }}\\Persistence\\Spy{{ domainEntity }}Query"
```

Comment:
Please try to remove all of the arguments here and add defaults to the new Spryk.